package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "react-project1",
+  "homepage": "https://sgiraldog.github.io/react-project1",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "chart.js": "^2.9.3",
+    "dotenv": "^8.2.0",
+    "gh-pages": "^3.1.0",
```

Comment:
not all dependencies are suppose to go here, there are a couple the project doesn't use that you use during develop, gh-pages is an example of one of them, these kind of dependencies should be dev-dependencies